script/db_migrate

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+db=$1
+migrations=$(ls db/migrations/*.up.sql)
```

Comment:
there's actually a tool we've been using to manage this: https://github.com/golang-migrate/migrate